web/shared/src/typeahead.ts

```diff
@@ -44,6 +48,10 @@ type UnicodeEmoji = {
     emoji_code: string;
     reaction_type: "unicode_emoji";
     is_realm_emoji: false;
+    has_reacted: boolean;
```

Comment:
I suspect if `UnicodeEmoji `type contains these fields. Can you double check this?

Reply:
Ok. Then you should make this a prep-commit(expanding the type).

Reply:
Also its worth looking at `EmojiDict` type in `emoji.ts`; whether that is a better fit for `search_results` and other parameters in this module(`emoji_picker.ts`)

Reply:
> Also its worth looking at `EmojiDict` type in `emoji.ts`; whether that is a better fit for `search_results` and other parameters in this module(`emoji_picker.ts`)

I think the current type is the right one, as the `EmojiDict` type from `emoji.ts`, when modified, results in the error: `Error: Caller should pass in a single row.`.  



> Ok. It will be better to make this a prep-commit(expanding the type).

Sure!